src/api/search/types.ts

```diff
@@ -22,12 +23,17 @@ export interface Product {
     welcomePackageInfo?: any;
     liked: boolean;
     recommendationInfo?: any;
-    bundleDealInfo?: any;
+    bundleDealInfo?: {
+        bundleDealId: number;
+        bundleDealLabel: string;
+    };
     priceMaxBeforeDiscount: number;
     trackingInfo: {
-        multiSearchTracking: number[];
-        viralSpuTracking?: any;
-        businessTracking?: any;
+        multiSearchTracking?: number[];
```

Comment:
`multiSearchTracking` could sometimes be `null`:  
![image](https://user-images.githubusercontent.com/57709309/152377387-7af69dcb-711d-4047-99d2-fb80c7c7965c.png)